src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator/CalculateEngine.cs

```diff
@@ -34,6 +34,13 @@ public CalculateResult Interpret(string input, CultureInfo cultureInfo, out stri
                 return default;
             }
 
+            // check for division by zero
+            if (input.Contains("/0") || input.Contains("/ 0"))
```

Comment:
Replace with regexp `\/\s*0(?!\.)` to not block division by number with decimal digits and add test case. Explain regexp in comment.